etc/kayobe/kolla/globals.yml

```diff
@@ -37,7 +37,7 @@ glance_tag: 7.0.2.1-1
 haproxy_tag: 7.0.2.1-1
 heat_tag: 7.0.2.1-1
 horizon_tag: 7.0.2.1-1
-influxdb_tag: 7.1.0.1
+influxdb_tag: 7.0.2.1-1
```

Comment:
Does the image contain the new version? Has this come from the kolla update?

Reply:
When the image is built it will fetch the latest version from the repo automatically (1.7.6). Now that I think about it, I will pin it.